i18n/allLangs.json

```diff
@@ -36,9 +37,17 @@
 			"tr": "tr",
 			"turkish": "tr",
 			"turkçe": "tr",
-			"zh": "zh",
-			"chinese": "zh",
-			"中文": "zh"
+			"zh": "zh-hans",
+			"chinese": "zh-hans",
+			"中文": "zh-hans",
+			"zh-hans": "zh-hans",
+			"Chinese (Simplified)": "zh-hans",
+			"Simplified Chinese": "zh-hans",
+			"简体中文": "zh-hans",
+			"zh-hant": "zh-hant",
+			"Chinese (Traditional)": "zh-hant",
+			"Traditional Chinese": "zh-hant",
+			"繁體中文": "zh-hant"
```

Comment:
the keys should be lower case only. 

Feel free to add more possible names for the language. those keys are what users can enter to get the language.